great_expectations/datasource/fluent/snowflake_datasource.py

```diff
@@ -72,7 +101,8 @@ def validate_parts(cls, parts: Parts, validate_port: bool = True) -> Parts:
         if domain is None:
             raise _UrlDomainError()
 
-        return AnyUrl.validate_parts(parts=parts, validate_port=validate_port)
+        validated = AnyUrl.validate_parts(parts=parts, validate_port=validate_port)
+        return validated
```

Comment:
```suggestion
        return AnyUrl.validate_parts(parts=parts, validate_port=validate_port)
```
Revert this meaningless change